include/modem/lte_lc.h

```diff
@@ -234,6 +247,37 @@ int lte_lc_edrx_param_set(const char *edrx);
  */
 int lte_lc_edrx_req(bool enable);
 
+/**
+ * @brief Set the parameters for the default PDP context.
+ * Must be called prior to `lte_lc_init` or `lte_lc_init_and_connect`.
+ *
+ * @param[in]  type            One of enum lte_pdp_type
+ * @param[in]  apn             null terminated APN string
+ * @param[in]  ip4_addr_alloc  0 - Allocate IPV4 address via NAS signaling
+ *	(default), 1 - Allocate IPV4 address via DHCP.
+ * @param[in]  nslpi           0 - NSLPI value from configuration is used
+ *	(default), 1 - Value "Not configured".
+ * @param[in]  secure_pco      0 - Protected PCO transmission is not requested
+ *	(default), 1 - Protected PCO transmission is requested
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+int lte_lc_pdp_context_set(enum lte_lc_pdp_type type, const char *apn,
+			bool ip4_addr_alloc, bool nslpi, bool secure_pco);
```

Comment:
Please fix the alignment (you can adjust it with spaces, to match the opening brace).